code/game/gamemodes/cult/runes.dm

```diff
@@ -11,6 +11,7 @@
 	var/bcolor
 	var/strokes = 2 // IF YOU EVER SET THIS TO MORE THAN TEN, EVERYTHING WILL BREAK
 	var/cultname = ""
+	var/hint = "" // hint on how cultists can use this rune
```

Comment:
> Could this be moved to a codex category? 🥺

no one checks it anyway :trollface:

Reply:
(but if you give me documentation on how to use it i might :)) )

Reply:
> (but if you give me documentation on how to use it i might :)) )

Check the other codex categories, and just copy what they do, except with runes. Actually, it might be better to hold off on this until my other codex PR is done.

Reply:
ok then guess i can resolve this for now